config.h.in

```diff
@@ -1,6 +1,8 @@
 #ifndef _OPENSCAP_CONFIG_H_
 #define _OPENSCAP_CONFIG_H_
 
+#include "compat.h"
```

Comment:
shoudn't you do all the cmakedefines and then include compat.h? otherwise the header won't work if I am reading this correctly